src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -370,7 +371,7 @@ MultiPlanTree(Query *queryTree)
  * more functionality in our distributed planning.
  */
 static void
-ErrorIfQueryNotSupported(Query *queryTree)
+ErrorIfQueryNotSupported(Query *queryTree, bool subqueryPushdown)
```

Comment:
We talked with @mtuncer in person and decided to move this check to #1289 on `ErrorIfCannotPushdownSubquery()` function.